Sources/NIOIRC/IRCMessageParser.swift

```diff
@@ -135,13 +136,37 @@ public struct IRCMessageParser {
       return String(data: Data(slice), encoding: .utf8) // Sigh, the pain.
     }
 
+    /* parse metadata */
+
+      let metadataSource : Swift.Slice<UnsafeRawBufferPointer>?
```

Comment:
Consistent indent w/ the rest would be nice :-) (I don't personally care too much about indentation styles, but they should be consistent within a project, which is 2-space right now in NIOIRC)